docs/tutorial/part-zero/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Programming Basics (Part Zero)
+typora-copy-images-to: ./
+---
+
+Here’s a list of basic programming knowledge that is necessary before moving on in the Gatsby tutorials. If you are new to any of these things, we've provided links to helpful tutorials on these subjects!:
+* *Command Line:* The command line is a text interface you can open on your computer. It allows you to run commands to your computer's operating system and it's necessary to learn this skill to become a programmer. [Codecademy's Command Line tutorial](https://www.codecademy.com/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command) for Mac and Linux users, and [this tutorial](https://www.computerhope.com/issues/chusedos.htm) for Windows users. Even if you are a Windows user, the first page of the Codecademy tutorial is still worth reading because it explains _what_ the command line is, not just how to use it.
+* *Code editor:* Code editing programs allow you to write, edit, and save the code you'll need to write to make your website work. [VS Code Studio](https://code.visualstudio.com/download) is a great code editor for Mac, Linux, and Windows users. Refer to their docs for help on getting started. There are many other great code editors; you may want to ask other programmers what code editor they prefer.
+* *Browser console:* Any internet browser you use has a console that allows you to inspect what is happening the information on each web page. This will help you figure out how to solve errors that happen while you build your website. If you are using Chrome as your web browser, here is an explanation of [how the Chrome browser console works](https://developers.google.com/web/tools/chrome-devtools/console/). If you use another web browser, search for instructions on how to access and use that browser's console.
+* *[React](https://reactjs.org/)* is a JavaScript library that Gatsby uses. React knowledge is helpful but not necessary. In fact, many people are using Gatsby as a way to learn React.
```

Comment:
*Prior React experience

Reply:
Hmmm I'm a bit iffy actually on that change.

Reply:
Either way. Both sound good to me.